### DISCUSSION TITLE
Making multiline cards

### DISCUSSION THREAD

**Discussion by debanjandhar12** [debanjandhar12/logseq-anki-sync#88] [Documentation]
Documentation has been moved from this location to this page: https://debanjandhar12.github.io/logseq-anki-sync/docs/card-types/multiline

**Comment by ghost** [comment]
Hi,
thank you very much for the great plug in.
I am not sure, if this is the right place to ask for an additional feature. 
It would be great, when using incremental multiline cards, that you could create flashcards, that ask you from top till bottom without showing the other bullet points.

Logseq:
What is? #cad #incremental
- A
- B
- C

=> Anki:
First card: Front: What is?
Back:
- A
- ?
- ?

Second card: Front: What is?
Back
- ?
- B
- ?

Third card: Front :What is?
Back
-?
-?
-C

That would be great to have. Thank you very much for the hard work. 🤓

**Comment by htigraine** [comment]
Hello, thank you so much for your work! I really appreciate the anki integration you made for logseq. 

Since the "specifying extra information" function is only available for single blocks, I was wondering in #incremental if there is a way to disable the flashcard option for a children block, i.e. it appears as extra information and is not considered as a children flashcard? 

I am just curious if this function existed/that there are plans to write this in future, please don't mind me if it is not within your plans to update it, or that if I came across as demanding. Thank you very much regardless, this plugin has been amazingly helpful.

**Reply by htigraine** [reply]
Requested in https://github.com/debanjandhar12/logseq-anki-sync/issues/204

**Comment by debanjandhar12** [comment]
An tiny update: card-group now applies tags like #reversed and #bidirectional on the children cards as well. You can use either of '#bidirectional'  or '#forward'  or '#reversed' to overwrite the direction for specific children cards too.

![image](https://github.com/debanjandhar12/logseq-anki-sync/assets/49021233/c3ecbd0d-6493-44ab-b952-4fdecf9c1ff4)

**Comment by debanjandhar12** [comment]
It is now possible to prevent syncing cards by using the using the `#no-anki-sync` tag:
![image](https://github.com/debanjandhar12/logseq-anki-sync/assets/49021233/354a7de6-7a76-4db9-8320-99bc13d98e37)

**Comment by KacperKazan** [comment]
Hi, thank you very much for this plugin!
Just a quick question on using the `#incremental` tag

in the example above, `#incremental` seems to incrementally hide one element whilst showing all the rest of the elements

how could I use this for sequential steps where I should only test my recall on the next step without getting hints about later steps?

for example if I have

1. step 1
2. step 2
3. step 3
4. step 4

and the generated card is testing my recall on 'step 2'
then is it possible to generate a card like

forward:
1. step 1
2. [...]
backward:
1. step 1
2. step 2
3. step 3
4. step 4

i.e. if testing 'step 2', the previous steps can be shown but the proceeding steps (3 & 4) should not be shown as they could give away too obvious hints about what 'step 2' is

I hope that makes sense?
